src/test/java/com/fcgl/madrid/repository/PostRepositoryIntegrationTest.java

```diff
@@ -1,4 +1,55 @@
 package com.fcgl.madrid.repository;
 
+import com.fcgl.madrid.MadridApplication;
+import com.fcgl.madrid.forum.model.InternalStatus;
+import com.fcgl.madrid.forum.model.PostRequest;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTestContextBootstrapper;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.*;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(SpringRunner.class)
+//@DataJpaTest
+@SpringBootTest(classes = MadridApplication.class, webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 public class PostRepositoryIntegrationTest {
+
+    @LocalServerPort
+    private int port;
+
+    TestRestTemplate restTemplate = new TestRestTemplate();
+
+    HttpHeaders headers = new HttpHeaders();
+
+    @BeforeClass
+    public static void start() {
+        Log logger = LogFactory.getLog(SpringBootTestContextBootstrapper.class);
+        logger.info("\n***********PostRepositoryIntegrationTest Running***********\n");
+    }
+
+    //TODO: Doesn't actually do what I want it to do... Testing is weird on this one
```

Comment:
Was not able to successfully get an Integration test running. 

A solution is to add a @ResponseBody to the Controller. But this makes curl requests not work. Need to find a work around